README.md

```diff
@@ -103,90 +106,65 @@ These instructions assume you've used the
+2. For Decred, [dcrd](https://github.com/decred/dcrd) and [dcrwallet](https://github.com/decred/dcrwallet), installed from the [v1.7.x release binaries](https://github.com/decred/decred-release/releases), or built from the `release-v1.7` branches. Alternatively you may use Decrediton or the dcrwallet application in SPV mode, or the native light wallet.
+3. For Litecoin, [Litecoin Core v0.21.x](https://litecoin.org/). Alternatively,
+you may use an external Electrum-LTC wallet with the name "default_wallet".
+4. For Dogecoin, [Dogecoin Core v1.14.5+](https://dogecoin.com/).
+5. For ZCash, [zcashd v5.1](https://z.cash/download/).
+6. For Bitcoin Cash, [Bitcoin Cash Node v24+](https://bitcoincashnode.org/en/)
 
 ### Connect Wallets and Register
 
-1. Start the client. `./dexc`
-2. In your browser, navigate to **localhost:5758**
+1. Start the client. Either go to the "DEX" tab within Decrediton, or with the
+   standalone client, open a command prompt in the folder containing the
+   pre-compiled dexc client files and run `./dexc` (`dexc.exe` on Windows).
+2. In your web browser, navigate to http://localhost:5758. Skip this step if
+   using Decrediton.
+
+[//]: # "TODO: either update or remove all of these screenshots for the current UI!"
 
 <img src="docs/images/omnibar-client.png" width="250">
 
 3. Create your **client application password**. You will use this password to perform all future security-sensitive client operations, including registering, signing in, and trading.
 
 <img src="docs/images/client-pw.png" width="250">
 
-4. Connect to your Decred wallet. The client will auto-fill most of your wallet settings, but you will need to specify the account name. If you haven't already, follow the instructions above to create a trading account. If you really, really want to trade on the default wallet account, the account name is `default`. Enter the wallet password, which is the password you set up with **dcrwallet**. Enter the app password you created in step 3.
+4. Connect to your Decred wallet. If using Decrediton, this will have been done for you The client will auto-fill most of your wallet settings, but you will need to specify the account name. If you haven't already, follow the instructions above to create a trading account. If you really, really want to trade on the default wallet account, the account name is `default`. Enter the wallet password, which is the password you set up with **dcrwallet**. Enter the app password you created in step 3.
 
 <img src="docs/images/decred-reg.png" width="250">
 
-5. Enter the dex address, probably **dex.decred.org**.
+5. Enter the dex address of your choice, probably **dex.decred.org**.
 
 <img src="docs/images/add-dex-reg.png" width="250">
 
 6. Check the registration fee, and enter your password one more time to authorize payment.
```

Comment:
For internal clients, you will need to send the fee first here to register. Or do we assume decrediton so this is too much info?

Reply:
All of these steps are outdated.  I kinda wanted to see what you guys thought about the screenshot removal idea, and then compress and update these steps for the simplified reg/setup workflow we have in this release.

Reply:
OK, agree. Updating the text of the steps now, and will gather screenshots after we review the text.